docs/topics/db/managers.txt

```diff
@@ -455,13 +455,11 @@ Throughout this section, we will use the term "automatic manager" to mean a
 manager that Django creates for you -- either as a default manager on a model
 with no managers, or to use temporarily when accessing related objects.
 
-Sometimes this default class won't be the right choice. One example is in the
-:mod:`django.contrib.gis` application that ships with Django itself. All ``gis``
-models must use a special manager class (:class:`~django.contrib.gis.db.models.GeoManager`)
-because they need a special queryset (:class:`~django.contrib.gis.db.models.GeoQuerySet`)
-to be used for interacting with the database.  It turns out that models which require
-a special manager like this need to use the same manager class wherever an automatic
-manager is created.
+Sometimes this default class won't be the right choice. For example, let's say 
+your Django application contains a library of genetic data. The default manager 
```

Comment:
Maybe we should just remove the "For example" stuff. It's not clear why genetic data would need a special queryset.